gui_interactions.md

```diff
@@ -0,0 +1,416 @@
+# GUI interactions
+
+This document is meant to inform and guide the development of various GUI
+interactions for Ignition Gazebo.
+
+## Overview
+
+Ignition Gazebo's graphical interface's high-level goals are to:
+
+1. **Introspect** a running simulation
+2. **Interact** with a running simulation
+3. **Edit** simulation worlds and robots
```

Comment:
Just to clarify, **Introspect** a log of a simulation that has been already performed in  past and is not running at the moment (for example because you are running a system that is quite slow and is not able to run in real time) it explicitly out of scope for Ignition Gazebo's GUI?

Reply:
@traversaro As far as I know, logging the state of the simulation is already possible (I opened https://github.com/robotology/gym-ignition/issues/32 some while ago to keep it in mind). I'm not sure that this is what you mean with your comment, and I cannot provide any information about it since I never tried the current logging capability of Ignition Gazebo and I'm not aware of its features / limitations. For sure either @chapulina or @mabelzhang can provide more details.

Reply:
As @diegoferigo mentioned, logging and playback is already supported and can be introspected from the GUI. [Here](https://ignitionrobotics.org/api/gazebo/3.0/log.html)'s a tutorial. That's what I meant by "recorded simulation". Let me know if you can think of a way to make that clearer.